ExportAdapter.js

```diff
@@ -34,8 +34,19 @@ ExportAdapter.prototype.connect = function() {
     return this.connectionPromise;
   }
 
+  var usernameStart = this.mongoURI.indexOf('://') + 3;
+  var lastAtIndex = this.mongoURI.lastIndexOf('@');
+  var encodedMongoURI = this.mongoURI;
+  var username = null;
+  var password = null;
```

Comment:
It does not look like `username` or `password` are used... and it looks like the database uri is being somewhat validated here, so how about we throw an error if it's not?